### PR TITLE
Remove cloud6 infrastructure

### PR DESCRIPTION
**Pull request by dirkmueller**
Cloud 6 was EOLed April 15th, 2018 so we could start
to remove the testing coverage for it to free up
CI ressources for newer products.

### CODE REVIEW

File: scripts/mkcloud [hunk 1]
```diff
@@ -1138,11 +1130,10 @@ Mandatory
     cloudpv=/dev/vdx (default /dev/vdb)
         Device where a LVM physical volume will be created, all data lost on this device
         Should be at least 80 GB. The volume group will be called 'cloud'.
-    cloudsource=develcloud6/7/8/9 | ocatacloud8 | susecloud9 | GM6 | GM6+up | GM7 | GM7+up | GM8 | GM8+up | M?  (default '')
+    cloudsource=develcloud7/8/9 | ocatacloud8 | susecloud9 | GM6 | GM6+up | GM7 | GM7+up | GM8 | GM8+up | M?  (default '')
```
Comment (nicolasbock) [inline scripts/mkcloud]: What about `GM6`?

File: scripts/mkcloud [hunk 2]
```diff
@@ -1385,7 +1376,7 @@ function sanity_checks
 
     if [ -z "$cloudsource" ] ; then
         echo "Please set the env variable:"
-        echo "export cloudsource=M?|develcloud6|develcloud7|develcloud8|develcloud9|susecloud9|GM6|GM6+up|GM7|GM7+up|GM8|GM+up"
```
Comment (nicolasbock) [inline scripts/mkcloud]: Ditto

File: scripts/qa_crowbarsetup.sh [hunk 3]
```diff
@@ -930,7 +913,7 @@ function onadmin_set_source_variables
             CLOUDLOCALREPOS="SUSE-OpenStack-Cloud-8-official"
         ;;
         *)
-            complain 76 "You must set environment variable cloudsource=develcloud6|develcloud7|develcloud8|Mx|GM6|GM7"
+            complain 76 "You must set environment variable cloudsource=develcloud7|develcloud8|Mx|GM6|GM7"
```
Comment (nicolasbock) [inline scripts/qa_crowbarsetup.sh]: Ditto

**Comment by rsalevsky** [comment]
Won't this break the Continues Delivery deployment option?

File: scripts/mkcloud (hunk 2 above)
Comment (dirkmueller) [reply scripts/mkcloud]: @nicolasbock done

**Comment by dirkmueller** [comment]
@rsalevsky are we still running ci for the continuous delivery option? where is it running? I've never found it

**Comment by jdsn** [comment]
> are we still running ci for the continuous delivery option? where is it running? I've never found it

The job `cloud-crowbar-testbuild-pr-trigger` is also triggering jobs for the sap-oc org. This is done due to the config in:
https://github.com/SUSE-Cloud/automation/blob/master/scripts/github_pr/github_pr_crowbar.yaml#L84-L94

**Comment by jdsn** [comment]
Please combine this PR with this one https://github.com/SUSE-Cloud/automation/pull/2585
Find out, what we need to keep and what parts can go.

In case we need to keep Cloud 6 CD then #2622 should not remove support for cloud6.

**Pull request closed without merging**